.github/workflows/c3-e2e.yml

```diff
@@ -75,6 +75,39 @@ jobs:
           git config --global user.email wrangler@cloudflare.com
           git config --global user.name 'Wrangler Tester'
 
+      - name: Deploy test worker
+        if: steps.changes.outputs.everything_but_markdown == 'true'
```

Comment:
We must skip this step if there is no `CLOUDFLARE_ACCOUNT_ID` set, since PRs from forks would fail.